cpp/Makefile

```diff
@@ -21,7 +21,7 @@ $(foreach t, $(slice_translators), $(eval $(call create-translator-project,$(cal
 
 include $(foreach p,$(load_first),src/$p/Makefile.mk)
 include $(filter-out $(foreach p,$(load_first),%/src/$p/Makefile.mk),$(shell find $(lang_srcdir)/src -name Makefile.mk))
-$(call make-projects,$(projects),make-cpp-src-project,,$(filter all cpp,$(ICE_BIN_DIST)))
+$(call make-projects,$(projects),make-cpp-src-project,,)
```

Comment:
Should we clean up make-projects to remove this last argument or is it used elsewhere?

Reply:
I would check in a follow up PR.